aeron-driver/src/main/java/io/aeron/driver/DriverConductor.java

```diff
@@ -1764,15 +1764,44 @@ private NetworkPublication newNetworkPublication(
         }
     }
 
+    private int getSndbufLength(final UdpChannelTransport channelEndpoint)
+    {
+        final int sndbufLength = channelEndpoint.socketSndbufLength();
+        if (sndbufLength == 0)
+        {
+            return ctx.osDefaultSocketSndbufLength();
+        }
+        else
+        {
+            return sndbufLength;
+        }
+    }
+
+    private int getRcvbufLength(final UdpChannelTransport channelEndpoint)
```

Comment:
See the above comment on `getSndbufLength`.

Reply:
Done